packages/integration-karma/test/misc/synthetic-shadow-bypass/Event.target.spec.js

```diff
@@ -0,0 +1,80 @@
+// Inspired from WPT:
```

Comment:
I don't understand why this test is needed. We are testing that the polyfill can be actually bypassed? Could you explain @ekashida?

Reply:
Yeah, I thought such a test might be useful since the difference between patching the prototype and patching the instance is subtle and we would otherwise not be able to catch a regression related to this approach. What do you think?

Reply:
If we really want to avoid regression in the future. I think the best approach here would be to check if the `target`, `currentTarget`, `path`, and `composedPath` are coming from the prototype and not from the instance.

```js
expect(event.hasOwnProperty('target')).toBeFalse();
expect(Object.getPrototypeOf(event).hasOwnProperty('target')).toBeTrue();
```